formstack-baton-requests/src/main/scala/com/gu/identity/formstackbatonrequests/aws/LambdaClient.scala

```diff
@@ -0,0 +1,34 @@
+package com.gu.identity.formstackbatonrequests.aws
+
+import com.gu.identity.formstackbatonrequests.circeCodecs._
+import com.amazonaws.services.lambda.AWSLambdaClient
+import com.amazonaws.services.lambda.model.{InvokeRequest, InvokeResult}
+import com.gu.identity.formstackbatonrequests.BatonModels.SarPerformRequest
+import com.gu.identity.formstackbatonrequests.SarLambdaConfig
+import com.typesafe.scalalogging.LazyLogging
+import io.circe.syntax._
+
+import scala.util.Try
+
+trait LambdaClient {
```

Comment:
This object and trait does not contain changes from https://github.com/guardian/identity-processes/pull/52/files, it's just showing up as a change because I've moved it into its own file